docs/examples/ex_jro_isr_beam.rst

```diff
@@ -0,0 +1,86 @@
+.. _ex-jro-beam-loc:
+
+Calculate JRO ISR Beam Location
+===============================
+
+For measurements made using a single beam, it may be more appropriate to account
+for the changes in beam direction with altitude when determining the measurement
+location instead of using the location of the radar.  The method
+:py:meth:`instruments.methods.jro.calc_measurement_loc` (see :ref:`meth-jro`)
+uses the beam azimuth and elevation measurements to determine the geodetic
+latitude and longitude.
+
+This method is designed to be used with the JRO ISR data, and so assumes the
+azimuths and elevations have data variable names with the format ``'eldir#'``
+and ``'azdir#'`` (where **#** is the beam number), or ``'elm'`` and ``'azm'``.
+It will modify the :py:attr:`pysat.Instrument.data` object by adding latitude
+(``'gdlat#'``) and longitude (``'gdlon#'``) variables for every beam that has
+appropriately labeled azimuth and elevatiton data.  If the azimuth and elevation
+angle variables don't specify the beam number, **#** will be set to ``'_bm'``.
+
+The easiest way to use :py:meth:`instruments.methods.jro.calc_measurement_loc`
+is to attach it to the JRO ISR :py:class:`pysat.Instrument` as a `custom
+function <https://pysat.readthedocs.io/en/latest/tutorial/tutorial_custom.html>`_
+before loading data.
+
+.. code::
+
+   import datetime as dt
+   import pysat
+   import pysatMadrigal as pysat_mad
+
+   jro_obl = pysat.Instrument(inst_module=pysat_mad.instruments.jro_isr,
+                              tag='oblique_long')
+   jro_obl.custom_attach(pysat_mad.instruments.methods.jro.calc_measurement_loc)
+
+
+If necessary, download the desired data before loading it.  The geographic
+beam locations will be present alongside the azimuths and elevations.
+
+.. code::
+
+   ftime = dt.datetime(2010, 4, 12)
+
+   if not ftime in jro_obl.files.files.index:
+       jro_obl.download(start=ftime)
+
+   jro_obl.load(date=ftime)
+   'gdlat_bm' in jro_obl.variables and 'gdlon_bm' in jro_obl.variables
```

Comment:
An extra line or comment may improve readability of this data check

Reply:
Some users could be new and things that are clear to us might not be for a new user. My initial reaction was that a bit of separation between the load statement and the data check may help, and a comment of, checking data, makes it clear at a glance.

Reply:
How about I move the download statement to the previous block?  Then there's even less separation between the data check and the sentence that explains the data check.

Reply:
I see the preceding paragraph talks about the calc_measurement_loc function. The sentence after the code refers to the above command should be true. A comment in the code that a data check is being performed makes it clear to new users or users who are skimming. My preference is really for most of the statements to have a comment, like the DMSP example code, even if it seems like the code alone is clear.